test/utilities/oas-validator.test.ts

```diff
@@ -225,14 +263,14 @@ describe('OasValidator', () => {
           description: 'a string',
         };
 
-        it('throws an error', () => {
-          expect(() =>
-            OasValidator.validateObjectAgainstSchema(null, schema, [
-              'body',
-              'facility',
-              'id',
-            ]),
-          ).toThrow(
+        it('returns a validation error', () => {
```

Comment:
```suggestion
        it('returns a validation failure', () => {
```

There are a few more of these below, might be nice to keep them consistent